platform.js

```diff
@@ -148,6 +148,32 @@ module.exports = {
       return module.exports.spawn("pkill -f " + path.join(module.exports.getInstallDir(), "chrome-linux") + "\n");
     }
   },
+  killExplorer() {
+    if (module.exports.isWindows() && module.exports.getWindowsVersion() !== "7") {
+      return module.exports.spawn("taskkill /f /im explorer.exe");
```

Comment:
@ahmedalsudani Yes, because it's used in a chain of promises (and the rest of methods with the same prefix work the same way). I don't see the advantage of removing the else clause (and, again, the other methods follow the same structure)